tests/TestPNL.py

```diff
@@ -1,55 +1,55 @@
 import os
 import sys
 
-sys.path.append("")
+BASE_DIR = os.path.join(os.path.dirname(__file__), '..')
+sys.path.append(BASE_DIR)
```

Comment:
don't do this --- check discussions in https://github.com/cmu-phil/causal-learn/pull/59